### ISSUE TITLE
node_modules ignore rule is raising false positive

### ISSUE THREAD

**Issue by jasonkarns** [SamVerschueren/clinton#91]
Getting this lint error:

```
  .gitignore
  ✖  node_modules is not being ignored. Add it to .gitignore.                                                          gitignore
```

But node_modules is properly ignored:

```
$ cat .gitignore 
/node_modules
```
